plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -387,6 +514,50 @@ function MenuDialog:init()
 
 end
 
+function MenuDialog.isValidModifier(number)
+    if tonumber(number) then
+        return true
+    else
+        return false
+    end
+end
+
+
+function MenuDialog.parseInputTime(str)
+    local num = tonumber(str:sub(1, -2))
+    local unit = str:sub(-1)
+    local totalMinutes
+    if unit == "m" then
+        totalMinutes = num
+    elseif unit == "h" then
+        totalMinutes = num * 60
+    elseif unit == "d" then
+        totalMinutes = num * 1440
+    else
+        error("Invalid unit!")
+    end
+    return totalMinutes
+end
+
+function MenuDialog.splitCommasToTable(input_string)
+    local t = {}
+    for w in string.gmatch(input_string, "[^,]+") do
+        local s = string.gsub(w, "%s", "")
+        table.insert(t, s)
+    end
+    return t
+end
+
+function MenuDialog.isIntervalStringValid(interval_string)
```

Comment:
What if user inputs 0m or empty string?

Reply:
Hmm, fair concern. I did do 0m manually previously and then intervals wouldn't increase. Perhaps could ask for minimally 1m interval, also checking empty strings is needed, right.